### ISSUE TITLE
CLDR import issue

### ISSUE THREAD

**Issue by srl295** [python-babel/babel#104]
Hello! I haven't looked into Babel for a few years, quite impressive changes.   I will add it to http://cldr.unicode.org

I noticed on -  https://github.com/mitsuhiko/babel/blob/master/scripts/import_cldr.py#L124 that `import_cldr.py` has hard coded file paths. Please don't assume certain file paths, they can and will change.

Please note for example http://unicode.org/repos/cldr/trunk/specs/ldml/tr35-info.html#Contents  : 

 _Future versions of CLDR may split the data in a different fashion. Do not depend on any specific XML filename or path for supplemental data._

**Comment by etanol** [comment]
Yes, all XML files in the `supplemental` directory have the same root element, and they will probably easily merged at the DOM level.
